src/server/routes/entity/author.js

```diff
@@ -96,11 +96,20 @@ router.get(
 	middleware.loadGenders, middleware.loadLanguages,
 	middleware.loadAuthorTypes, middleware.loadRelationshipTypes,
 	(req, res) => {
-		const {markup, props} = entityEditorMarkup(generateEntityProps(
+		const markupProps = generateEntityProps(
 			'author', req, res, {
 				genderOptions: res.locals.genders
 			}
-		));
+		);
+		markupProps.initialState.nameSection = {
+			disambiguation: '',
+			exactMatches: null,
+			language: null,
+			name: req.query.name,
```

Comment:
```suggestion
			name: req.query?.name ?? '',
```

This ensures there is a default empty string if query or query.name are undefined.
Without that, creating an entity without an initial name results in a server error in [isAliasEmpty](https://github.com/bookbrainz/bookbrainz-site/blob/88ce6fe4a61cf755ff14f5ad764d48a8070976cf/src/client/entity-editor/helpers.ts#L61) and breaking the page:
```
× TypeError: Cannot read property 'length' of undefined
     at isAliasEmpty (/home/bookbrainz/bookbrainz-site/lib/client/entity-editor/helpers.js:109:22)
     at NameSection.render (/home/bookbrainz/bookbrainz-site/lib/client/entity-editor/name-section/name-section.js:207:42)
```
     
The same applies for the other routes too.